AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1957,20 +1956,19 @@ protected void updateScreenCounts() {
             int newRevCount = 0;
             Sched sched = getCol().getSched();
             for (Long did : deckIds) {
-                youngRevCount += sched._youngRevCountForDeck(did);
+                youngRevCount += sched._criticalReviewCountForDeck(did);
                 learnRevCount += sched._lrnForDeck(did);
                 newRevCount += sched._newForDeck(did, sched._deckNewLimit(did));
             }
             int cardsLeft = youngRevCount + learnRevCount + newRevCount * 2;
             String cardsLeftPart;
             if (cardsLeft > 10) {
-                cardsLeftPart = String.format("%s critical", cardsLeft);
+                cardsLeft = (cardsLeft + 49) / 50 * 50;
+                cardsLeftPart = String.format("<%s critical", cardsLeft);
```

Comment:
would this show `<50 critical` for as long as you have any critical? Am I understanding this correctly?

Reply:
That's right, unless we have <=10 critical in which case it will continue to show the thumbsup (👍)